hw5-10_to_do_list/react_ts/src/components/TaskView/TaskView.tsx

```diff
@@ -0,0 +1,96 @@
+import "./TaskView.css";
+import Task from "../../types/Task";
+import { EntityID } from "../../repositories/Repository";
+import Button from "../Button/Button";
+import { datesAreEqual } from "../../utils/Utils";
+
+export default function TaskView({
+    isDisplayed,
+    task,
+    className,
+    onCheckboxClick,
+    deleteTask,
+}: {
+    isDisplayed: boolean;
+    task: Task;
+    className: string;
+    onCheckboxClick: (task: Task) => Promise<void>;
+    deleteTask: (taskId: EntityID) => Promise<void>;
+}) {
+    const formatDate = (dateLikeString: string) => {
```

Comment:
It was okey to keep this out of a component

Reply:
fixed